base/memory/shared_memory_tracker.cc

```diff
@@ -117,16 +103,33 @@ void SharedMemoryTracker::DecrementMemoryUsage(
   usages_.erase(mapping.raw_memory_ptr());
 
 #if defined(CASTANETS)
-  auto it = mappings_.find(mapping.guid());
-  if (it != mappings_.end()) {
-    mappings_.erase(it);
-    VLOG(2) << "Del mapping" << mapping.guid()
-            << " num: " << mappings_.size();
-  }
+  RemoveMapping(mapping.guid(), mapping.raw_memory_ptr());
 #endif
 }
 
 #if defined(CASTANETS)
+void SharedMemoryTracker::AddMapping(
+    const UnguessableToken& guid, size_t size, void* ptr) {
+  auto it = mappings_.find(guid);
+  if (it == mappings_.end()) {
+    scoped_refptr<CastanetsMemoryMapping> castanets_mapping =
+        CastanetsMemoryMapping::Create(guid, size);
+    castanets_mapping->AddMapping(ptr);
+    mappings_.emplace(guid, castanets_mapping);
+  } else
+    it->second->AddMapping(ptr);
```

Comment:
In case of chromium coding style, one of if else phrase has its parentheses, the others have their own parentheses also.

'git cl format' command will do that for you I guess.

Reply:
I've reformatted this commit by using git cl format.